solutions/baseline/main.tf

```diff
@@ -0,0 +1,81 @@
+#######################################################################################################################
+# Resource Group
+#######################################################################################################################
+module "resource_group" {
+  source                       = "terraform-ibm-modules/resource-group/ibm"
+  version                      = "1.1.6"
+  resource_group_name          = var.use_existing_resource_group == false ? ((var.prefix != null && var.prefix != "") ? "${var.prefix}-${var.resource_group_name}" : var.resource_group_name) : null
+  existing_resource_group_name = var.use_existing_resource_group == true ? var.resource_group_name : null
+}
+
+########################################################################################################################
+# OCP VPC cluster (single zone)
+########################################################################################################################
+
+locals {
+  prefix = var.prefix != null ? (var.prefix != "" ? var.prefix : null) : null
+}
+
+data "ibm_is_subnets" "vpc_subnets" {
+  vpc = var.vpc_id
+}
+
+locals {
+  cluster_vpc_subnets = {
+    for subnet in data.ibm_is_subnets.vpc_subnets.subnets :
+    "default" => [{
+      id         = subnet.id
+      zone       = subnet.zone
+      cidr_block = subnet.ipv4_cidr_block
+    }]
+  }
```

Comment:
Did we deep dive on this? This is going to do a data lookup against the VPC and automatically add all subnets to the cluster right? Do we need to give consumer more control over this?